src/auth/auth.ts

```diff
@@ -822,7 +824,7 @@ export class TenantAwareAuth
 export class Auth extends BaseAuth<AuthRequestHandler>
   implements FirebaseServiceInterface, AuthInterface {
 
-  public INTERNAL: AuthInternals = new AuthInternals();
+  public INTERNAL: AuthInternals;
```

Comment:
The reason for this change was that typescript enforces that `super()` must be the first call in the constructor if you have any inline-initialized fields.  But I have to construct the `HttpClient` before `super()`